protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -18,71 +18,61 @@
 
 <?php $this->beginContent('@admin/views/group/_manageLayout.php', ['group' => $group]) ?>
 <div class="panel-body">
-    <?php $form = ActiveForm::begin(['acknowledge' => true]) ?>
-    <?= $form->field($group, 'name')->textInput(['readonly' => !$canManage]) ?>
-    <?= $form->field($group, 'description')->textarea(['rows' => 5, 'readonly' => !$canManage]) ?>
-
-    <?php if ($canManage) : ?>
-        <?= $form->field($group, 'type')->dropDownList($group::getTypeOptions()) ?>
-        <?= $form->field($group, 'subgroups')->widget(GroupPicker::class, ['groupType' => $group::TYPE_NORMAL]) ?>
-        <?= $form->field($group, 'parent')->widget(GroupPicker::class, ['groupType' => $group::TYPE_SUBGROUP]) ?>
-    <?php endif ?>
+    <?php $form = ActiveForm::begin(['acknowledge' => true, 'renderOnlySafeFields' => true]) ?>
```

Comment:
maybe `renderOnlySafeAttributeFields` would be a good naming?

Reply:
@luke- I think better name is `renderOnlyActiveAttributes` because parent Yii model has methods as `isAttributeActive()` and `activeAttributes()` which return a result depending on scenarios.